src/components/MatrixBuilder/index.js

```diff
@@ -0,0 +1,57 @@
+import "./MatrixBuilder.scss"
+
+export const MatrixBuilder = ({ data, setData }) => {
+	const createMatrix = () => {
+		let arr = new Array();
+		for (let i = 0; i < data.rows; i++) {
+			arr[i] = new Array();
+			for (let j = 0; j < data.columns; j++) {
+				arr[i][j] = Math.floor(Math.random() * (999 - 100) + 100);
+			}
+		}
+		setData(prevState => ({
+			...prevState,
+			arr: arr,
+		}))
+	}
+
+	const enterData = (event) => {
```

Comment:
never use class name or element as checking value. just create handlers for the every input